### PR TITLE
Mock performance.now(), fixes #24

### LINKED ISSUE #24: This library should also mock `window.performance.now`
See https://developer.mozilla.org/en-US/docs/Web/API/Performance/now

### PR DESCRIPTION
**Pull request by insidewhy**

### CODE REVIEW

**Comment by insidewhy** [comment]
It might be nice to provide an API to advance the `performance.now()` time beyond the `now()` time in order to test certain edge conditions, but this PR is probably a good start.

File: src/index.js [hunk 1]
```diff
@@ -13,9 +13,11 @@ const dateClass = mockDateClass(Date);
 if (global.window) {
   // dom env
   global.window.Date = dateClass;
+  global.window.performance.now = function() { return dateClass.now(); };
 } else {
   // node / native env
   global.Date = dateClass;
+  require('perf_hooks').performance.now = function() { return dateClass.now(); };
```
Comment (hustcc) [inline src/index.js:20]: what is perf_hooks?

Comment (insidewhy) [reply src/index.js:20]: It's a performance measurement API built into node's API. It's essentially node's version of the browser's monotonic clock (window.performance.now).

When mocking the non-monotonic clock I think it's unnatural for the monotonic clock to be free.

**Comment by hustcc** [comment]
> It might be nice to provide an API to advance the `performance.now()` time beyond the `now()` time in order to test certain conditions for users of `window.performance.now`, but this PR is probably a good start.

Also, document of readme should be update.
